### PR TITLE
Convert ANGSD doHaploCall output to fasta format

### LINKED ISSUE #30: Convert ANGSD doHaploCall output to fasta format
Script from Tom (Edana)

### PR DESCRIPTION
**Pull request by verku**
The *haplo.gz files are converted to fasta format per sample, printing every genome position to preserve genome coordinates

### CODE REVIEW

File: read-harvester/bin/haplo2fasta.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
+from sys import argv
+from itertools import izip, islice
+import gzip
+
+def parse_haplo(haplo_file):
+
+  samples = ["SF-1_42","AI-1_119","BL-11_9","KU-23_3",
```
Comment (bilalbioinfo) [inline read-harvester/bin/haplo2fasta.py]: The samples names are hard-coded in this python code!!

Comment (verku) [reply read-harvester/bin/haplo2fasta.py]: This is fixed in the updated script

File: read-harvester/bin/haplo2fasta.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+from sys import argv
+from itertools import islice
+import gzip
+
+def parse_haplo(haplo_file, samplename):
```
Comment (verku) [inline read-harvester/bin/haplo2fasta.py:7]: Here you can see the updated code without hard-coded sample names. It is supposed to run now for one sample at a time (as it is done in the pipeline)

File: read-harvester/bin/haplo2fasta.py (hunk 1 above)
Comment (verku) [reply read-harvester/bin/haplo2fasta.py]: See below or here: https://github.com/NBISweden/LTS-L_Dalen_2302/pull/38/files#r1766485763

**Pull request merged**